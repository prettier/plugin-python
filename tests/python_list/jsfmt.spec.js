run_spec(__dirname, ["python"], { pythonVersion: "2", trailingComma: "none" });
run_spec(__dirname, ["python"], { pythonVersion: "2", trailingComma: "all" });
run_spec(__dirname, ["python"], { pythonVersion: "3" });
