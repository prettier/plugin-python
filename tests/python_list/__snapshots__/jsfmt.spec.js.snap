// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list.py 1`] = `
a = [
    "a",
    'b'
]

a = [
    "a",
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'hhhhhhhhhhhhhh',
]

a = ['aaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'bbbbbbbbbbbbbbbbbbbbbbbbbbbb', 'cccccccccccccccccccccccccccc', 'dddddddddddddddddddddddddddd']
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a = ["a", "b"]

a = ["a", "b", "c", "d", "e", "f", "g", "hhhhhhhhhhhhhh"]

a = [
    "aaaaaaaaaaaaaaaaaaaaaaaaaaaa",
    "bbbbbbbbbbbbbbbbbbbbbbbbbbbb",
    "cccccccccccccccccccccccccccc",
    "dddddddddddddddddddddddddddd"
]

`;

exports[`list.py 2`] = `
a = [
    "a",
    'b'
]

a = [
    "a",
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'hhhhhhhhhhhhhh',
]

a = ['aaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'bbbbbbbbbbbbbbbbbbbbbbbbbbbb', 'cccccccccccccccccccccccccccc', 'dddddddddddddddddddddddddddd']
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a = ["a", "b"]

a = ["a", "b", "c", "d", "e", "f", "g", "hhhhhhhhhhhhhh"]

a = [
    "aaaaaaaaaaaaaaaaaaaaaaaaaaaa",
    "bbbbbbbbbbbbbbbbbbbbbbbbbbbb",
    "cccccccccccccccccccccccccccc",
    "dddddddddddddddddddddddddddd",
]

`;

exports[`list.py 3`] = `
a = [
    "a",
    'b'
]

a = [
    "a",
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'hhhhhhhhhhhhhh',
]

a = ['aaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'bbbbbbbbbbbbbbbbbbbbbbbbbbbb', 'cccccccccccccccccccccccccccc', 'dddddddddddddddddddddddddddd']
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a = ["a", "b"]

a = ["a", "b", "c", "d", "e", "f", "g", "hhhhhhhhhhhhhh"]

a = [
    "aaaaaaaaaaaaaaaaaaaaaaaaaaaa",
    "bbbbbbbbbbbbbbbbbbbbbbbbbbbb",
    "cccccccccccccccccccccccccccc",
    "dddddddddddddddddddddddddddd"
]

`;
